packages/app/src/specs/banners/TrackedBanner.cy.tsx

```diff
@@ -126,5 +126,15 @@ describe('<TrackedBanner />', () => {
         cy.get('@recordEvent').should('not.have.been.called')
       })
     })
+
+    context('when eventData is undefined', () => {
+      it('should not record event', () => {
+        cy.mount({
+          render: () => <TrackedBanner bannerId="test-banner" hasBannerBeenShown={true} eventData={undefined} />,
```

Comment:
To mimic the scenario being used in this PR, you would want to test with `hasBannerBeenShown` equal to false, correct?

```suggestion
          render: () => <TrackedBanner bannerId="test-banner" hasBannerBeenShown={false} eventData={undefined} />,
```